### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by williamh890**
Update docs

This pr is build on-top of #160

### CODE REVIEW

File: docs/_static/instance-processing.rst [hunk 1]
```diff
@@ -19,21 +19,21 @@ will call the science code to do the actual processing needed to generate an
 image. The orchestration code is always the same, regardless of the process
 which is being run.
 
+Science Code
+------------
+
+This can be pretty anything as long as it provides a compatible interface.
+A python file called ``hyp3_handler.py`` containing a function called ``handler``
```
Comment (Askaholic) [inline docs/_static/instance-processing.rst]: Need `handler` function arguments or link to page which describes function and it's arguments.

File: docs/_static/setup.rst [hunk 2]
```diff
@@ -24,15 +24,30 @@ Authorize an email address in SES
...
 
+Earthdata Credentials
+^^^^^^^^^^^^^^^^^^^^^
+
+    To download data within hyp3, an earthdata account is needed.
+    To create an earthdata account go to:
+
+        - `Register for Earthdata`_
+
+    **WARNING:** Make sure that all the required EULA's are excepted on your
```
Comment (Askaholic) [inline docs/_static/setup.rst]: Maybe "Note" and not "warning"

File: docs/_static/setup.rst [hunk 3]
```diff
@@ -24,15 +24,30 @@ Authorize an email address in SES
...
+    Earthdata account or else hyp3 will not be able to successfully download
```
Comment (Askaholic) [inline docs/_static/setup.rst]: 'HyP3' not 'hyp3'

File: docs/_static/setup.rst [hunk 4]
```diff
@@ -24,15 +24,30 @@ Authorize an email address in SES
...
+    data.
+
+    The username and password for this account will be used later when
+    creating the CloudFormation stack.
+
 Launching the CloudFormation stack
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
    Once you have a key pair, and have verified an email with SES, your ready to
```
Comment (Askaholic) [inline docs/_static/setup.rst]: 'you're' not 'your'

File: docs/_static/setup.rst [hunk 5]
```diff
@@ -42,7 +57,8 @@ Launching the CloudFormation stack
 Stack Parameters
 ~~~~~~~~~~~~~~~~
 
-    The ``authorized email`` and ``ec2-key-pair`` from the previous steps will be used here.
+    The ``authorized email``, ``ec2-key-pair`` and ``earhtdata credentials`` from the
+    previous steps will be used here.
 
     All the parameters that can have default values can be left unchanged.
```
Comment (Askaholic) [inline docs/_static/setup.rst]: All the parameters that ~~can~~ have default values can be left unchanged.

Remove can.

File: docs/_static/setup.rst [hunk 6]
```diff
@@ -53,6 +69,10 @@ Stack Parameters
     username will be used for accessing with the hyp3. The hyp3 email can be different from the verified
```
Comment (Askaholic) [inline docs/_static/setup.rst]: HyP3

Comment (Askaholic) [reply docs/_static/setup.rst]: I'd just do a find and replace on the whole docs directory

Comment (Askaholic) [reply docs/_static/setup.rst]: Just make sure you do it on whole word matches only

File: lambdas/scheduler/src/schedule/job.py [hunk 7]
```diff
@@ -1,25 +1,32 @@
-from typing import NamedTuple
+from typing import NamedTuple, Union
 
-from hyp3_events import NewGranuleEvent, StartEvent, EmailEvent
+from hyp3_events import NewGranuleEvent, StartEvent, EmailEvent, Hyp3Event
 from hyp3_db.hyp3_models import Subscription, User, Process
 
 
 class Job(NamedTuple):
+    """ All the required information for scheduling and creating a hyp3 job """
```
Comment (Askaholic) [inline lambdas/scheduler/src/schedule/job.py]: HyP3

File: modules/hyp3_process/src/hyp3_process/handler/handler.py [hunk 8]
```diff
@@ -30,6 +30,42 @@
 def make_hyp3_processing_function_from(
     handler_function: HandlerFunction
 ) -> ProcessingFunction:
+    """ Convertes a **handler function -> process function**
+
+        A handler function is a function to run arbitrary processing
```
Comment (Askaholic) [inline modules/hyp3_process/src/hyp3_process/handler/handler.py]: ~~to run~~ which runs

File: modules/hyp3_process/src/hyp3_process/handler/handler.py [hunk 9]
```diff
@@ -30,6 +30,42 @@
...
+        code and is not connected to HyP3. Handler functions are responsible
+        for download any data they require for processing. This includes
```
Comment (Askaholic) [inline modules/hyp3_process/src/hyp3_process/handler/handler.py]: Handler functions are responsible for download ~~any data they require for processing~~ their own data.

File: modules/hyp3_process/src/hyp3_process/handler/handler.py [hunk 10]
```diff
@@ -30,6 +30,42 @@
...
+        granules and DEM files.
```
Comment (Askaholic) [inline modules/hyp3_process/src/hyp3_process/handler/handler.py]: Maybe there could be a shameless plug for the granule util here :P

File: modules/hyp3_process/src/hyp3_process/process.py [hunk 11]
```diff
@@ -15,10 +15,25 @@
 
 
 class Process:
-    def __init__(self, *, handler_function) -> None:
-        self.add_handler(handler_function)
+    """
+    Wraps process hyp3 processing functionality around a lambda function
```
Comment (Askaholic) [inline modules/hyp3_process/src/hyp3_process/process.py]: HyP3

File: modules/hyp3_process/src/hyp3_process/handler/handler.py [hunk 12]
```diff
@@ -30,6 +30,42 @@
...
+
+        .. code-block:: python
+
+           def handler_function(
```
Comment (Askaholic) [inline modules/hyp3_process/src/hyp3_process/handler/handler.py]: Should link to here from instance-processing.rst

File: docs/_static/modules/hyp3_process.rst [hunk 13]
```diff
@@ -0,0 +1,30 @@
+.. _hyp3_process:
+
+Module: Hyp3 Process
```
Comment (Askaholic) [inline docs/_static/modules/hyp3_process.rst]: HyP3

**Review by Askaholic — CHANGES_REQUESTED** [review]

File: docs/_static/setup.rst (hunk 2 above)
Comment (williamh890) [reply docs/_static/setup.rst]: I was debating that.

**Pull request merged**